pkgs/by-name/ha/hanko/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, fetchFromGitHub, installShellFiles, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hanko";
+  version = "0.5.2";
+
+  src = fetchFromGitHub {
+    owner = "SRv6d";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-gytnUta/sNa8vJMvVS/DMwRc4R/8rmOnEfBKEjRpfGs=";
+  };
+  cargoHash = "sha256-rlKFS1ppUoyZ1X2PeG8tEB44Ysqi7kCLqR+ECVIdPjs=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installManPage assets/manpages/*.1
+
+    installShellCompletion assets/completions/hanko.bash
+    installShellCompletion assets/completions/_hanko
+    installShellCompletion assets/completions/hanko.fish
+  '';
+
+  meta = with lib; {
```

Comment:
Avoid with lib;
```suggestion
  meta = {
```

Reply:
I saw this commonly done in other package files, is there some sort of style guide that advocates against this?

Reply:
More detailed discussion can be found at https://github.com/NixOS/nixpkgs/issues/208242 — it's a style lint done for new packages but it hasn't been implemented for older packages yet

Reply:
My bad, didn't realize I wasn't actually using it at all, thanks for the link.

Reply:
*was needed but fixed it